internal/controller/cluster/rack.go

```diff
@@ -1507,7 +1498,7 @@ func (r *SingleClusterReconciler) isVolumeAttachmentAddedOrUpdated(
 		if volumeMount != nil {
 			// Found, check for updated
 			if getOriginalPath(volumeMount.MountPath) != attachment.Path ||
-				volumeMount.ReadOnly != attachment.ReadOnly ||
+				volumeMount.ReadOnly != ptr.Deref(attachment.ReadOnly, false) ||
```

Comment:
We use a wrapper of this func called `GetBool` to avoid passing `false` at multiple places.  Check at other places as well